active/PagerControl/PagerControl.md

```diff
@@ -0,0 +1,356 @@
+| PrefixText | * Text displayed before the editable ComboBox indexing component. <br><br> ![image](https://user-images.githubusercontent.com/16964652/49760540-93759880-fc79-11e8-9c4f-3c3222b95a13.png) |
+| NumberOfPages | * When a total number of indices (N) is given, this suffix string will appear after the editable ComboBox indexing component and read "of N". Localization will put "N" where it should be in a given language. <br><br> ![image](https://user-images.githubusercontent.com/16964652/49760557-9a9ca680-fc79-11e8-943f-3425d993fdeb.png) |
+
+## API Details
+<!-- The exact API, in MIDL3 format (https://docs.microsoft.com/en-us/uwp/midl-3/) -->
+
+```c++ 
+enum PagerDisplayMode
+{
+    Auto,
+    ComboBox,
+    NumberBox,
+    ButtonPanel,
+};
+
+
+enum ButtonVisibilityMode
+{
+    Auto,
+    AlwaysVisible,
+    HiddenOnLast,
+    None,
+};
+
+runtimeclass PagerControlPageChangedEventArgs
+{
+    Integer CurrentPage{get; }:
+}
+
+runtimeclass PagerControl
+{
+    PagerControl();
+
+    PagerDisplayMode Display;
+    
+    Integer NumberOfPages;
+    
+    ButtonVisibilityMode FirstButtonVisibility;
+    ButtonVisibilityMode PreviousButtonVisibility;
+    ButtonVisibilityMode NextButtonVisibility;
+    ButtonVisibilityMode LastButtonVisibility;
+
+    FirstButtonCommand="FirstButtonPressedEvent"
+    PreviousButtonCommand="PreviousButtonPressedEvent"
+    NextButtonCommand="NextButtonPressedEvent"
+    LastButtonCommand="LastButtonPressedEvent"
+    PagerInputCommand="PagerInputEvent"
+    
+    IconSource FirstButtonGlyph;
+    IconSource PreviousButtonGlyph;
+    IconSource NextButtonGlyph;
+    IconSource LastButtonGlyph;
+    
+    String FirstButtonText;
+    String PreviousButtonText;
+    String NextButtonText;
+    String LastButtonText;
+
+    Windows.UI.Xaml.Input.ICommand FirstButtonCommand;
+    Windows.UI.Xaml.Input.ICommand PreviousButtonCommand;
+    Windows.UI.Xaml.Input.ICommand NextButtonCommand;
+    Windows.UI.Xaml.Input.ICommand LastButtonCommand;
+    Windows.UI.Xaml.Input.ICommand PagerInputCommand;
+    
+    Windows.UI.Xaml.Style FirstButtonStyle;
+    Windows.UI.Xaml.Style PreviousButtonStyle;
+    Windows.UI.Xaml.Style NextButtonStyle;
+    Windows.UI.Xaml.Style LastButtonStyle;
+    
+    
+    Integer NumberOfIndicesShowing;
+    Boolean EllipsisEnabled;
+    Boolean EllipsisShowFirstAndLast;
+    Integer EllipsisMaxBefore;
+    Integer EllipsisMaxAfter;
+    Integer SelectedIndex;
+    
+    String PrefixText;
+    String SuffixText;
```

Comment:
Same here, should we provide localized default texts for those?

Reply:
Yes if these have default text, that should be localized..

Reply:
Yes the default text should be in the API notes section. PrefixText = "Page" and SuffixText = "of (NumberOfPages)". If NumberOfPages is blank, the SuffixText will be empty.